src/common/POLFields/FieldsLocation/FieldsLocation.js

```diff
@@ -28,7 +28,16 @@ import {
 
 const NO_VALIDATE = [];
 
-const FieldsLocation = ({ locations, disabled, isDisabledToChangePaymentInfo, withValidation }) => {
+const FieldsLocation = ({
+  changeLocation,
+  disabled,
+  isDisabledToChangePaymentInfo,
+  locationIds,
+  locations,
```

Comment:
`locations` might be missing, I would not render this component before that, since it's required in `FieldLocation`

Reply:
fixed